src/components/button/Button.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import './Button.css';
+
+const Button = ({fruitName, setFruitName, name}) => {
+    return (
+        <article className="card">
+            <h2>{name}</h2>
+            <button onClick={() => setFruitName(fruitName - 1)}
+                    disabled={fruitName === 0}>
+                -
+            </button>
+            {fruitName}
+            <button
+                onClick={() => setFruitName(fruitName + 1)} >
```

Comment:
Nu zal telkens wanneer `fruitName` veranderd een onnodige rerender van deze functie plaatsvinden. Wat je zou kunnen doen is de huidige waarde gebruiken die je vanuit de setFruitName functie al krijgt:
```suggestion
                onClick={() => setFruitName(currentValue => currentValue + 1)} >
```